target/linux/ramips/image/mt7621.mk

```diff
@@ -1679,6 +1679,19 @@ define Device/mikrotik_routerboard-m33g
 endef
 TARGET_DEVICES += mikrotik_routerboard-m33g
 
+define Device/mofinetwork_mofi5500-5gxelte
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 27656k
+  DEVICE_VENDOR := MoFi Network
+  DEVICE_MODEL := MOFI5500-5GXeLTE
+  DEVICE_PACKAGES := kmod-mt7615-firmware kmod-usb3 kmod-scsi-core kmod-leds-gpio \
+	kmod-wwan kmod-gpio-button-hotplug kmod-hsdma-mtk kmod-mmc kmod-mhi-bus \
+	kmod-mhipci-generic kmod-sdhci kmod-sdhci-mt7620 kmod-usb-core kmod-usb2 \
+	kmod-mac80211 kmod-mt7615e
```

Comment:
seems excessive.  Are all these extra packages required for basic device operation?  Note that due to the way initramfs images are built shared by the builders, these packages will not necessarily end up in all the "official" images.